README.md

```diff
@@ -1,2 +1,21 @@
 # comp110-worksheet-B
 Base repository for COMP110 worksheet B
+
+```Password = The chosen password that the user must find
+UserChosenWord = the word the user clicked/chose
+SameLetters = 0
+numbLetrs = the number of letters in Password
+
+If UserChosenWord =! Password
+	print "Entry denied"
+	for each letter in UserChosenWord and each letter in Password
+		if letter == letter in password
+			SameLetters++
+			
+	print SameLetters " / " numbLetrs
+else 
+	YouWinFunction
+```
+
+
+![Image of FlowChart](https://i.imgur.com/vdhSBUX.png)
```

Comment:
Flowchart is mostly OK, though "Setup game" is vague -- what steps does this involve? Also better to formulate loops using explicit counter variables and decision nodes rather than "After doing this 4 times", as this type of thing could be difficult to follow in a more complex flowchart